### PR TITLE
ArangoGraph Notebooks update

### PR DESCRIPTION
**Pull request by nerpaula**
preview: https://deploy-preview-1176--zealous-morse-14392b.netlify.app/docs/3.10/arangograph/notebooks.html#arangograph-magic-commands

### CODE REVIEW

File: arangograph/notebooks.md [hunk 1]
```diff
@@ -59,3 +69,83 @@ at anytime. The notebook's phase is updated accordingly.
...
+
+## ArangoGraph Magic Commands
+
+A list of the available magic commands you can interact with.
+Single line commands have `%` prefix and multi-line commands have `%%` prefix.
+
+**Database Commands**
+
+- `%listDatabases` - lists the databases on the database server.
+- `%whichDatabase` - which database am I connected to?
```
Comment (cw00dw0rd) [inline arangograph/notebooks.md:92]: Having a question here doesn't seem right. How about `returns the database name.`

File: arangograph/notebooks.md [hunk 2]
```diff
@@ -13,17 +13,27 @@ This documentation describes the beta version of the Notebooks feature and is
 subject to change. The beta version is free for all.
 {% endhint %}
 
-The ArangoGraph Insights Platform provides an easy way to run Jupyter Notebooks in the cloud,
-without having to move your data outside of the platform.
+The ArangoGraph Notebook is a JupyterLab notebook embedded in the ArangoGraph
+ecosystem. The notebook integrates seamlessly with the ArangoGraph ecosystem,
```
Comment (Simran-B) [inline arangograph/notebooks.md:17]: It's officially a platform, so the full name could be used here:
```suggestion
Insights Platform. The notebook integrates seamlessly with the ArangoGraph Insights Platform,
```

**Comment by cw00dw0rd** [comment]
@nerpaula we may make some function name changes to better align with the db module function names. I will report back once we have confirmation.

File: arangograph/notebooks.md [hunk 3]
```diff
@@ -13,17 +13,27 @@ This documentation describes the beta version of the Notebooks feature and is
...
+- How can I access certain documents?
+- How do I create a graph?
+and many more.
+
+The ArangoGraph Notebook also pre-installs `python-arango` and ArangoML connectors
+to PyG, DGL, CuGraph, as well as the FastGraphML library, so you can get started
+right away accessing data in ArangoDB to develop GraphML models using your
+favorite GraphML libraries with GPU's.
 
 ## How to create a new notebook
```
Comment (ansoboleva) [inline arangograph/notebooks.md:38]: please replace the screenshot with an updated version after the wording change

File: arangograph/notebooks.md [hunk 4]
```diff
@@ -59,3 +69,83 @@ at anytime. The notebook's phase is updated accordingly.
...
+
+## Getting Started notebook
+
+To get a better understanding of how to interact with your ArangoDB database
+cluster, use the ArangoGraph Getting Started template.
+The ArangoGraph Notebook automatically connects to the ArangoDB service
+endpoint, so you can immediately start interacting with it.
+
+1. Log in into the notebook you have created.
+2. Open the file browser and navigate to the `jovyan` directory.
```
Comment (ansoboleva) [inline arangograph/notebooks.md:81]: What's this step about? Don't see it on the UI. The Getting started notebook is just displayed in the file tree.

Comment (nerpaula) [reply arangograph/notebooks.md]: At the time I opened this PR, the Getting Started template was "hidden" under the jovyan directory, not really in plain sight. Hence, this step. 
Now, looks like the template is displayed in the file tree right after opening the notebook. I removed the step.

**Review by ansoboleva — APPROVED** [review]
lgtm

**Pull request merged**